### ISSUE TITLE
New Template:Text Classification in TF

### ISSUE THREAD

**Issue by oscar-defelice** [jrieke/traingenerator#13]
If you guys think there might be an interest in that I would like to offer my contribution for a text classifier in TensorFlow.

**Comment by jrieke** [comment]
@oscar-defelice Yes that would be fantastic! @themrcesi is already working on a text classification template in scikit-learn (#14) if you want to look for inspiration. Feel free to open a WIP PR as soon as you start :)
